### ISSUE TITLE
pulp server -> broken because of "--require-path=../"

### ISSUE THREAD

**Issue by rvion** [purescript-contrib/pulp#238]
```shell
pulp server --main main
* Server listening on http://localhost:1337/

* Invalid option `--require-path=../'

Usage: psc [FILE] [-o|--output ARG] [--no-tco] [--no-magic-do] [--no-opts]
           [-v|--verbose-errors] [-c|--comments] [--source-maps] [--dump-corefn]
           [-p|--no-prefix] [--json-errors]

Hash: 49d762b1c3f8bdedd964
Version: webpack 1.13.2
Time: 415ms
     Asset     Size  Chunks             Chunk Names
    app.js  1.77 kB       0  [emitted]  main
app.js.map  1.61 kB       0  [emitted]  main
chunk    {0} app.js, app.js.map (main) 31 bytes [rendered]
    [0] ./src/.webpack.js 31 bytes {0} [built] [1 error]

```

**Comment by hdgarrood** [comment]
Are you using an old version of pulp? This should be fixed as of 10.0.

**Comment by rvion** [comment]
indeed, thanks ! 

before:
```
pulp --version
Pulp version 9.0.1
psc version 0.10.2 using /Users/rvion/.local/bin/psc
```

```
npm i -g pulp
```

after:
```
pulp --version
Pulp version 10.0.0
psc version 0.10.2 using /Users/rvion/.local/bin/psc

pulp server
* Server listening on http://localhost:1337/
* Building project in /Users/rvion/dev/ps-test
* Build successful.
* Bundling JavaScript...
* Bundled.
```
